docs/user/managing-devices.md

```diff
@@ -547,6 +547,79 @@ spec:
 > [!NOTE]
 > Inline compose applications can have at most two paths. The first should be named `podman-compose.yaml`, and the second (override) must be named `podman-compose.override.yaml`.
 
+### Adding Application Volumes
+
+> [!NOTE]
+> This feature requires the Flight Control Agent to run with **Podman version 5.5 or higher**.
+
+Applications can declare persistent data volumes that are populated from OCI artifacts. This allows delivering large datasets (such as ML models or static assets) as part of the application deployment.
+
+Volumes are declared under each application in the `volumes` field. These volumes are mounted into the application containers via the Compose `volumes` section.
+
+#### Specifying Volumes
+
+Each volume definition includes:
+
+| Field | Description |
+| ----- | ----------- |
+| `name` | Logical volume name. Must match the volume name referenced in the Compose file. |
+| `image.reference` | Fully qualified OCI artifact reference containing the volume contents. |
+| `image.pullPolicy` | (Optional) Defines pull behavior: `Always` or `IfNotPresent` (default: `IfNotPresent`). |
```

Comment:
just wanted to confirm that we don't want to include the `Never` policy option here.